forte/data/converter/converter.py

```diff
@@ -12,39 +12,101 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from typing import List, Tuple, Any, Optional
-
 import torch
 from torch import Tensor
 
+from forte.common import ValidationError
 from forte.data.converter.feature import Feature
 
 
 class Converter:
     """
-    This class has the functionality of converting a batch of Feature to a
-    PyTorch tensor. It will also do the padding for the given batch of data.
+    This class has the functionality of converting a batch of
+    :class:`forte.data.converter.Feature` to a PyTorch `Tensor`. It will also
+    do the padding for the given batch of :class:`forte.data.converter.Feature`.
+
+    Args:
+        default_dtype: the `dtype` of resulted tensor that will be obtained via
+            converting the input features.
     """
 
     def __init__(self, default_dtype: torch.dtype = torch.long):
         self._default_dtype = default_dtype
 
     def convert(self, features: List[Feature]) -> \
             Tuple[Tensor, List[Tensor]]:
+        # pylint: disable=line-too-long
```

Comment:
it is better not disabling this, which line is unavoidable?

Reply:
Just the docstring. Any other way?

Reply:
Since the new committers of the repo will try to copy what the repo is doing. I try not to set an unwanted precedent. Can we add additional linebreaks in the docstring?

Reply:
That's OK but just then the rendered code block is so clean. But I think that's fine, right?

Reply:
If the rendered code block indent just like correct python code, it should be fine.

Reply:
Done